src/components/PlayerView/index.tsx

```diff
@@ -35,9 +35,25 @@ export interface PlayerViewProps extends BasePlayerViewProps, PlayerViewEvents {
    */
   player: Player;
 
+  /**
+   * The `FullscreenHandler` that is used by the `PlayerView` to control the fullscreen mode.
+   */
   fullscreenHandler?: FullscreenHandler;
 
+  /**
+   * The `CustomMessageHandler` that can be used to directly communicate with the embedded WebUi.
+   */
   customMessageHandler?: CustomMessageHandler;
+
+  /**
+   * Can be set to `true` to enter fullscreen mode, or `false` to exit fullscreen mode.
+   * Should not be used to get the current fullscreen state. Use `onFullscreenEnter` and `onFullscreenExit`
+   * or the `FullscreenHandler.isFullscreenActive` property to get the current state.
```

Comment:
Isn't this surprising and error prone? How about making it a method call (eg `requestFullscreen(Boolean)`) (no getter) or renaming it `isFullscreenRequested`.

Reply:
We were advised to go for properties instead of methods. Having a method is also not directly possible in this setup. We were discussing that a bit and this is what we settled on.
Yea, naming this a bit different sounds like a good idea.

Reply:
https://github.com/bitmovin/bitmovin-player-react-native/pull/169/commits/14271faf65c61ef661030b787a8b35fc7dc31538